pootle/apps/pootle_project/models.py

```diff
@@ -30,11 +30,12 @@
 from pootle.core.cache import make_method_key
 from pootle.core.mixins import CachedTreeItem
 from pootle.core.models import VirtualResource
-from pootle.core.url_helpers import (get_editor_filter, get_path_sortkey,
-                                     split_pootle_path, to_tp_relative_path)
```

Comment:
well from debugging this code, right now, im really failing to see the benefit of adding extra code, to do something that python does perfectly well.